src/formulations/AV/av_formulation.cpp

```diff
@@ -189,5 +191,8 @@ AVEquationSystem::AddKernels()
       da_dt_name,
       _v_name,
       std::make_shared<hephaestus::VectorFEWeakDivergenceKernel>(vector_fe_weak_divergence_params));
+
+  logger.info("AVOperator AddKernels: {} seconds", sw);
```

Comment:
Correction  - "AVEquationSystem::AddKernels" - another possible place where fetching the class (and possibly method) name explicitly could help clarity

Reply:
Done!